crates/sui-core/src/overload_monitor.rs

```diff
@@ -204,6 +211,37 @@ fn check_overload_signals(
     (overload_status, load_shedding_percentage)
 }
 
+// Return true if we should reject the txn with `tx_digest`.
+fn should_reject_tx(
+    load_shedding_percentage: u32,
+    tx_digest: TransactionDigest,
+    minutes_since_epoch: u64,
+) -> bool {
+    let mut hasher = XxHash64::with_seed(minutes_since_epoch);
```

Comment:
can you add a TODO for adding a secret salt? The simplest thing I can think of would be to use the hash of the first consensus commit of the epoch (and a dummy value before that is available).

We can't keep it a guaranteed secret (byzantine validator can always disclose it) but it is at least something of an obstacle.